appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -144,6 +148,27 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
                 size: 'P3v2',
                 family: 'Pv2',
                 capacity: 1
+            },
+            {
```

Comment:
Actually, sorry, but I just realized that we may not want to show all of these Pxv3 plans in advanced creation. Could you move these plans into another function that is used specifically for jboss eap apps? Something like `getPremiumV3Skus`?

Reply:
@nturinski thanks for your message and really sorry for the late response, i've updated the pr that only show v3 pricing for jboss eap